src/openfermion/resource_estimates/pbc/thc/compute_lambda_thc.py

```diff
@@ -0,0 +1,137 @@
+# coverage: ignore
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+from typing import Tuple
+import numpy as np
+import numpy.typing as npt
+
+from openfermion.resource_estimates.pbc.thc.thc_integrals import (
+    KPTHCDoubleTranslation,)
+from openfermion.resource_estimates.pbc.hamiltonian import (
+    HamiltonianProperties,)
+
```

Comment:
is it worth having a dummy `HamiltonianProperties` descendant for thc for consistency? and making the base class abstract?

Reply:
Not sure if abstract base class is necessary? I added a properties dataclass for THC